mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3063,10 +3114,10 @@ async fn handle_timed_out_taker_orders(ctx: MmArc, ordermatch_ctx: &OrdermatchCo
             .await
             .ok();
         let maker_order: MakerOrder = order.into();
-        {
-            let mut my_maker_orders = ordermatch_ctx.my_maker_orders.lock();
-            my_maker_orders.insert(uuid, Arc::new(AsyncMutex::new(maker_order.clone())));
-        }
+        ordermatch_ctx
+            .makerorders_ctx
+            .lock()
```

Comment:
@artemii235 here's the lock inside the loop

Reply:
It's not a problem here as Taker->Maker order conversion is not that frequent :)